spec/views/idv/shared/_ssn.html.erb_spec.rb

```diff
@@ -0,0 +1,119 @@
+require 'rails_helper'
+
+describe 'idv/shared/_ssn.html.erb' do
+  include Devise::Test::ControllerHelpers
+
+  let(:proofing_device_profiling_collecting_enabled) { nil }
+  let(:lexisnexis_threatmetrix_account_id) { 'test_org_id' }
+  let(:session_id) { 'ABCD-1234' }
+  let(:updating_ssn) { false }
+  let(:js_domain) { 'h.online-metrix.net' }
+
+  let(:tags_js_url) {
+    "https://#{js_domain}/fp/tags.js?org_id=#{lexisnexis_threatmetrix_account_id}&session_id=#{session_id}"
+  }
+
+  let(:tags_iframe_url) {
+    "https://#{js_domain}/fp/tags?org_id=#{lexisnexis_threatmetrix_account_id}&session_id=#{session_id}"
+  }
+
+  let(:script_tag_rendered) {
+    expect(rendered).to have_css("script[src='#{tags_js_url}']", visible: false)
+  }
+
+  let(:noscript_tag_rendered) {
+    expect(rendered).to have_css("noscript iframe[src='#{tags_iframe_url}']", visible: false)
+  }
+
+  let(:session_id_input_rendered) {
+    expect(rendered).
+      to have_css(
+        "input[type=hidden][name='doc_auth[threatmetrix_session_id]'][value='#{session_id}']",
+        visible: false,
+      )
+  }
+
+  let(:script_tag_not_rendered) {
+    expect(rendered).not_to have_css("script[src*='#{js_domain}']", visible: false)
+  }
+
+  let(:noscript_tag_not_rendered) {
+    expect(rendered).not_to have_css("noscript iframe[src*='#{js_domain}']", visible: false)
+  }
+
+  let(:session_id_input_not_rendered) {
+    expect(rendered).
+      not_to have_css('input[name="doc_auth[threatmetrix_session_id]"]', visible: false)
+  }
```

Comment:
ditto, regular helper method:
```suggestion
  def expect_session_id_input_not_rendered
    expect(rendered).
      not_to have_css('input[name="doc_auth[threatmetrix_session_id]"]', visible: false)
  end
```